zerver/lib/storage.py

```diff
@@ -40,12 +57,16 @@ def hashed_name(
             # use a no-op hash function for these already-hashed
             # assets.
             return name
-        if name in SYSTEM_BOTS_AVATAR_FILES.values():
+        if name.startswith(STATIC_AVATARS_DIR):
```

Comment:
I simplified the logic to hash these avatar URLs, if it's in the `static/images/static_avatars/` dir, it'd get hashed. I wonder if the naming for this folder makes sense, I didn't use disk_avatar or avatar_disk to avoid confusion with `avatar_disk_path()` function in testing..

Reply:
Yeah that seems cleaner.